### ISSUE TITLE
Support for Dead Letter Queueus in Consumers?

### ISSUE THREAD

**Issue by hassanzia32** [apache/pulsar-client-node#201]
Does the nodeJS client currently support [dead letter queues](https://pulsar.apache.org/docs/en/concepts-messaging/#dead-letter-topic)?

**Comment by nkurihar** [comment]
Unfortunately the node.js client doesn't support the dead letter topic for now.

**Comment by hassanzia32** [comment]
@nkurihar, thank you for confirming. Is there a timeline on when this would be added, if it exists?

Thanks for all your contributions to this client.

**Comment by nkurihar** [comment]
>  Is there a timeline on when this would be added, if it exists?

No, unfortunately we have not decided the timeline to add this feature yet...
